src/platforms/ctxlink/WiFi_Server.c

```diff
@@ -531,6 +536,12 @@ static void app_wifi_callback(uint8_t msg_type, void *msg)
 		break;
 	}
 
+	case M2M_WIFI_CONN_INFO_RESPONSE_EVENT: {
+		tstrM2MConnInfo *connection_info = (tstrM2MConnInfo *)msg;
+		memcpy(&conn_info, connection_info, sizeof(tstrM2MConnInfo));
```

Comment:
For what it's worth, `memcpy()` takes a void pointer for the pointer parameters so you could simplify this to `memcpy(&conn_info, msg, sizeof(tstrM2MConnInfo));` and forgo the pointer conversion and local.

Reply:
Agreed, however the local aids with debugging, that is why it was added.